config/deployer.sample.json

```diff
@@ -5,7 +5,17 @@
   "AWSAMI": "ami-003d3d03cfe1b0468",
   "ClusterName": "loadtest",
   "ClusterVpcID": "",
-  "ClusterSubnetID": "",
+  "ClusterSubnetIDs": {
+    "Agent": "",
+    "App": "",
+    "Database": [],
+    "ElasticSearch": "",
+    "Job": "",
+    "Keycloak": "",
+    "Metrics": "",
+    "Proxy": "",
+    "Redis": []
```

Comment:
Why do we need a list for Redis when it's just a single server? Contrary, why we _don't_ need a list for other resources like app/agent/proxy?

Reply:
I could use lists and then index by `count`, but I favoured simplicity in this first PR. if you think that's necessary it's a easy change.

Reply:
Went ahead and did the change: 46be0e8d9d7e64f71b55dd68d69a19803cb4bcf5

Reply:
Ah no no, my point was if Redis is a single server, then we don't need a list there. Unless the list is mandatory somehow.

Reply:
I've made the list change and if we only need one subnet (because there's only one server) we can just set only one subnet in the setting (or leave it empty to use the default from `aws_subnet.selected`). Feels simpler if every setting works the same way.

Reply:
Ok, conceptually it just felt odd to have an array when it's just one resource. But not a big deal.

Reply:
maybe if we use `aws_elasticache_replication_group` in the future?